tracardi/service/setup/setup_indices.py

```diff
@@ -106,11 +106,21 @@ async def remove_alias(alias_index):
             if index.aliased is False:
                 target_index = index.get_index_alias()
 
+            # Handles case when index exists
             if not await es.exists_index(target_index):
 
-                # Creates index and alias in one shot.
-
                 mapping = map['template'] if index.multi_index else map
+                exists_index_with_alias_name = await es.exists_index(alias_index)
+
+                # some app (for example a bridge) made Tracardi write to alias before creating indices
+                if exists_index_with_alias_name:
+                    logger.info(f"There exists index with name {alias_index}, which should be its alias name. "
+                                f"Data is going to be re-indexed and new index will be created.")
+
+                    if "aliases" in map:
+                        del map["aliases"]
+
+                # Creates index and alias in one shot if alias not popped from mapping variable
                 result = await es.create_index(target_index, mapping)
```

Comment:
This code will fail here. As stated in error log. 
```
File "/home/risto/PycharmProjects/tracardi-api/app/api/install_endpoint.py", line 71, in install
    result = {"created": await create_indices(), 'admin': False}
  File "/home/risto/PycharmProjects/tracardi/tracardi/service/setup/setup_indices.py", line 119, in create_indices
    raise ConnectionError(
ConnectionError: Index `072-dev.57a8e.tracardi-source` was NOT CREATED. The following result was returned: {'error': {'root_cause': [{'type': 'invalid_alias_name_exception', 'reason': 'Invalid alias name [57a8e.tracardi-source], an index exists with the same name as the alias', 'index_uuid': 'Hkr2pIy6RzWPTBNBmUHFXQ', 'index': '57a8e.tracardi-source'}], 'type': 'invalid_alias_name_exception', 'reason': 'Invalid alias name [57a8e.tracardi-source], an index exists with the same name as the alias', 'index_uuid': 'Hkr2pIy6RzWPTBNBmUHFXQ', 'index': '57a8e.tracardi-source'}, 'status': 400}
```

So this PR will never work. Can't reindex if can't create the proper set of alias and index.